checks.py

```diff
@@ -446,3 +448,29 @@ def authoritative_check(self):
                 authoritative_data['authoritative_old'] = '\' \''
 
         self.results_dict.update(authoritative_data)
+
+    def visibility_check(self):
+        '''
+        Make sure item's default visibility is set to True
+
+        Update results_dict with results for this item:
+                {'visibility_fix':''}
+        '''
+
+        fix_visibility = {'visibility_fix':'N'}
+
+        for layer in self.item.layers:
+            
+            #: Check if default vis is true; wrap in try/except for robustness
+            try:
+                properties = json.loads(str(layer.manager.properties))
+            except:
+                properties = None
+
+            if properties and not properties['defaultVisibility']:
+                self.set_visibility = True
+
+        if self.set_visibility:
+            fix_visibility = {'visibility_fix':'Y'}
```

Comment:
**suggestion**: the dictionary is already created, might as well update it instead of recreate it.

```suggestion
            fix_visibility['visibility_fix'] = 'Y'
```